lib/index.js

```diff
@@ -101,7 +101,9 @@ internals.validateRoutes = (server) => {
  * @param h Hapi Reponse Toolkit https://hapijs.com/api#response-toolkit
  */
 internals.onPreHandler = async (request, h) => {
-
+  //#28 server.binds send to h.context now
+  const config = h.context.config;
```

Comment:
Do you know what version of hapi this started in? Gotta be careful to not break backwards compatibillity.

Reply:
Think it was 17, was working with 18 but a previous report says it started on 17